.travis.yml

```diff
@@ -12,8 +12,8 @@ jobs:
     - php: 7.1
     - php: 7.2
     - php: 7.3
+    - php: 7.4
       env: WITH_COVERAGE=true
```

Comment:
I guess it's better to collect coverage on PHP 7.4 instead of PHP 7.3 - it's probably faster, too.